src/main/java/com/programmers/springbootbasic/domain/customer/JdbcCustomerRepository.java

```diff
@@ -1,25 +1,75 @@
 package com.programmers.springbootbasic.domain.customer;
 
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
 import org.springframework.stereotype.Repository;
 
+import java.util.Collections;
+import java.util.Map;
 import java.util.Optional;
+import java.util.UUID;
 
 @Repository
 public class JdbcCustomerRepository implements CustomerRepository {
-    // TODO:
-    //  5차에 구현 예정
+    private static final String CUSTOMER_ID_FOR_DATABASE = "customer_id";
+    private static final String CUSTOMER_ID_FOR_OBJECT = "customerId";
```

Comment:
```suggestion
    private static final String DATABASE_CUSTOMER_ID = "customer_id";
    private static final String CUSTOMER_ID = "customerId";
```

변수명 혹은 클래스명에 전치사를 사용하는 것을 그리 선호하지 않는 편입니다. 
이런 방식은 어떤가요??(그냥 의견을 물어보는거에요 ㅎㅎ)

Reply:
좀 이상하게 지었나 생각하긴 했는데 for object가 디폴트다 보니 for object를 지우는게 좀 더 좋은 것 같습니다!